medusa/cassandra_utils.py

```diff
@@ -483,7 +504,7 @@ def is_node_up(config, host):
     if int(config.cassandra.is_ccm) == 1:
         args = ['ccm', 'node1', 'nodetool']
     else:
-        args = ['nodetool', '-h', host]
+        args = Nodetool(config.cassandra).nodetool + ['-h', host]
```

Comment:
Is there a reason not to use the standard `self._nodetool.nodetool` here? (like for lines above) as we have:

```
self._nodetool = Nodetool(cassandra_config)
```

Reply:
The method `is_node_up` is not member of `Cassandra` class. So I can't access to `self` here

Reply:
oh! I remember that. We never fixed it... It should be part of Cassandra, but then for now this is probably good enough.

Reply:
ok, thanks for the review @arodrime